cmd/zed/dev/compile/command.go

```diff
@@ -200,6 +210,20 @@ func (c *Command) writeDAG(seq dag.Seq) {
 	}
 }
 
+func (c *Command) writeDescribe(ctx context.Context, seq dag.Seq, lk *lake.Root) {
+	info, err := describe.Analyze(ctx, seq, data.NewSource(nil, lk), nil)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+	out, err := json.MarshalIndent(info, "", "  ")
```

Comment:
Nit: Be consistent with `normalize`.
```suggestion
	out, err := json.MarshalIndent(info, "", "    ")
```